packages/login/src/components/StyledLoginForm.js

```diff
@@ -26,15 +28,36 @@ export const StyledLoginFormInputWrapper = styled.div`
   }
 `
 
-export const StyledLoginFormWrapper = styled.div`
+export const StyledLoginButton = styled(Button)`
+  &&&& {
```

Comment:
The loading icon should be scaled as well 
![image](https://user-images.githubusercontent.com/7300481/69349649-634fbb80-0c78-11ea-82f0-06e23ea222f4.png)